styles.css

```diff
@@ -315,4 +314,86 @@
   .btn:disabled {
     background-color: #2a2b33;
   }
+
+  .contact-form .flex-container-column,
+  .flex .flex-container-column,
+  .footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .contact-form {
+    margin-top: 75px;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 40px;
+    gap: 24px;
+  }
```

Comment:
Please avoid to use duplicate Duplicate class.
Class .contact-form is declared twice. margin-top declared twice. The last declaration is in effect 40px